camayoc/tests/qpc/cli/utils.py

```diff
@@ -34,12 +34,12 @@ def clear_all_entities():
     assert errors == [], output
 
 
-def wait_for_scan(scan_job_id, status="completed", timeout=900):
+def wait_for_scan(scan_job_id, status="completed", timeout=1200):
```

Comment:
How do you feel about using [`camayoc.scan_timeout`](https://github.com/quipucords/camayoc/blob/main/camayoc/config.py#L21) here?

That would lower it to 10 minutes by default, but then we will gain ability to set timeout through setting file or environment variable. So each pipeline could specify how much time it needs. We can also increase default in `config.py` if these tests need that much time.

Could be we have tried that in the past and it won't work due to some problem. I don't remember if we did.

Reply:
Yes, I do believe it is a good idea and I'm going to implement this way. Maybe laziness is the reason in the past.